### DISCUSSION TITLE
After update to 2.1.0rc1, launching MA still takes me to Settings page, not Home page.

### DISCUSSION THREAD

**Discussion by harryfine** [music-assistant/support#2624] [Q&A]
Thank you for the addon release candidate for MA. 

Unfortunately I still have the issue where launching the MA addon using version: 2.1.0rc1 takes me to the Settings page instead of the Home page.

**Comment by OzGav** [comment]
Let us know if it is still a problem with rc2

**Reply by OzGav** [reply]
If it is please confirm exactly which music providers and player providers you have installed. Feel free to post a screenshot of the providers settings page

**Reply by harryfine** [reply]
Just installed rc2.  Rebooted.  Confirmed new version of addon was installed.  Clicked on tab on sidebar and took me directly to Settings page.  I also have it in a panel view on a Webpage card.  It does the same thing. 

Also it seems to flash for a second before going to the settings page, as if it's trying to load the Home page, can't, and goes to the settings page.

Same issue.  
![ma1](https://github.com/user-attachments/assets/b234bc0f-ec0f-4f2b-adf9-3d56b861543d)
![ma2](https://github.com/user-attachments/assets/12b2767d-8df0-4cc6-96ff-f1fb5f190979)

**Reply by OzGav** [reply]
Thanks this will get looked at soon. There is an open issue so follow along there https://github.com/music-assistant/hass-music-assistant/issues/2491
